### ISSUE TITLE
Aktywacja konta 

### ISSUE THREAD

**Issue by mleczoo** [allegro/allegro-api#9361]
Czy mógłbym prosić o aktywację konta patrykba2024@gmail.com ?
![aktywacjakontasandbox](https://github.com/allegro/allegro-api/assets/174992063/42347e3e-466d-48f3-82a2-f63975f5c602)

**Comment by MartaNowaczyk** [comment]
Widzę, że posiadasz konto zwykłe, natomiast aby móc testować proces wystawiania ofert musisz mieć konto firmowe. Możesz przekształcić obecne konto korzystając z [tej zakładki](https://allegro.pl.allegrosandbox.pl/moje-allegro/moje-konto/dane-konta).

Następnie skonfiguruj proszę jeszcze [wypłaty](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/ustaw-wyplaty) i skorzystaj z naszego symulatora płatności w celu potwierdzenia.

**Comment by mleczoo** [comment]
Zmieniłem już na konto firmowe, ale link z konfiguracją wypłaty jest
nieprawidłowy.

pon., 8 lip 2024 o 12:52 MartaNowaczyk ***@***.***>
napisał(a):

**Comment by MartaNowaczyk** [comment]
Zmieniłeś na konto firmowe, jednak wybrałeś opcję ze sprzedażą okazjonalną - proszę przejdź na pełną weryfikację bym mogła aktywować w pełni to konto. Zrobisz to m.in. w zakładce [Środki i historia operacji](https://allegro.pl.allegrosandbox.pl/moje-allegro/sprzedaz/srodki-i-historia-operacji).

**Comment by mleczoo** [comment]
Czyli musze dodać konto bankowe i wykonać przelew ? Dopiero wtedy konto
będzie aktywne ?

pon., 8 lip 2024 o 13:21 MartaNowaczyk ***@***.***>
napisał(a):

**Comment by MartaNowaczyk** [comment]
tak, ale przelew jest fikcyjny, a numer konta podajesz z generatora kont bankowych.

**Comment by mleczoo** [comment]
To już zrobiłem

pon., 8 lip 2024 o 14:41 MartaNowaczyk ***@***.***>
napisał(a):

**Comment by MartaNowaczyk** [comment]
Nie widzę byś zrobił pełną aktywację. Możesz spróbować inną ścieżką. Zobacz czy w [ustawieniach konta](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/warunki-sprzedazy) masz link na dole "[WRÓĆ DO WYBORU WERYFIKACJI](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/przeslij-dokumenty)", jak tak to skorzystaj z niego i wybierz "Weryfikacja pełna". Uzupełnij dane i prześlij dowolne pliki, abym mogła aktywować to konto.

**Comment by mleczoo** [comment]
Czy możesz zobaczyć teraz ?

**Comment by MartaNowaczyk** [comment]
Teraz ok, aktywowałam twoje konto :)

**Comment by twozniakatcomp** [comment]
Dzień dobry, proszę o aktywację konta o loginie at_beactivepl

**Comment by MaciejFrackowiak** [comment]
@twozniakatcomp Konto aktywowane.

**Comment by twozniakatcomp** [comment]
> @twozniakatcomp Konto aktywowane.

dziękujemy serdecznie :)
